### DISCUSSION TITLE
Switch from Mistral to Levante

### DISCUSSION THREAD

**Discussion by denizural** [esm-tools/esm_tools#605] [HPCs]
This is a test question.
When will DKRZ switch from Mistral to Levante?

**Comment by seb-wahl** [comment]
I don't think this is a test question: I always thought, mistral will be decommissioned by the end of March but given the fact that the transfer of data from work from Mistral to levante hasn't started yet, I assume mistral will be around for longer. But I don't know an exact date.

**Reply by denizural** [reply]
Thanks for the feedback @seb-wahl. Due to the category limitation in GitHub discussions, I moved all HPC related questions / announcements here and made a test question just to show the use of `labels` to filter different HPCs. 
I also agree that the actual migration date is very important.

**Comment by fernandadialzira** [comment]
Can we already at least transfer the content of our home folder to Levante? According to DKRZ's e-mail the automatic transfer will only be done in the work folder and we need to transfer whatever we find important from our local home by ourselves. I wonder if I can start doing it already

**Comment by seb-wahl** [comment]
Yes, this can be done already. If you still work on mistral, you can always update your home on Levante from Mistral or vice-versa via rsync. If you have never used rsync before, there are tons of introductions out there on the web. Or any of the ESM-Tools developers will be able to help.

**Reply by pgierz** [reply]
Briefly, in case anyone looks here and is too lazy to google afterwards:
```
$ rsync -avzhP <source_location> <user@host:destination_location>
```

For the flags:
```
-a: all files
-v: verbose
-z: compress during transfer (this makes it a bit faster)
-h: use human-readable file sizes when reporting
-P: show progress information
```

One import caveat: `rsync` is sensitive to trailing slashes:

`/some/path/like/this`: that will copy across the folder
`/some/path/like/this/`: that will copy across the contents of the folder

**Comment by mandresm** [comment]
I think this link provides valuable information about it https://docs.dkrz.de/doc/levante/index.html
